docs/rules/require-tagless-components.md

```diff
@@ -0,0 +1,66 @@
+## Components must not have a wrapper element
+
+### Rule name: `require-tagless-components`
+
+Ember allows you to disable the wrapper element on a component (turning it into a "tagless" component). This is now the preferred style and the _only_ style allowed with Glimmer components.
+
+Instead of having the wrapper element implicitly defined by the component, all DOM elements should be represented in the component's template.
```

Comment:
Can you add a bullet point list which includes possible solutions to fix violations from this rule?
* Set `tagName: ''`
* Switch to a glimmer component
* Move the implicit wrapper `<div>` to the component template if still needed